### ISSUE TITLE
Add Indiana hypso data

### ISSUE THREAD

**Issue by lindsayplatt** [USGS-R/lake-temperature-model-prep#132]
Gretchen found this source: https://hub.arcgis.com/datasets/0ac283034cde4bb9bf8955eb5551be30_0

Seems like it would be similar to how NDGF data was parsed & added.

**Comment by jread-usgs** [comment]
Nice! Seems IN includes the NHDHR id as one of the fields of this dataset. So 
```r
mutate(site_id = paste0('nhdhr_', NHDID_WATE))
```
results in matches to 148 `site_id` values in the `1_crosswalk_fetch/out/canonical_lakes_sf.rds`
and there are 164 unique lake names in this database. The three that don't have `NHDID_WATE` values are CEDARVILLE RESERVOIR, DEAM LAKE, and HUNTINGBURG LAKE

out of the three, CEDARVILLE RESERVOIR looks like a river
![image](https://user-images.githubusercontent.com/2349007/78883116-a7bcd780-7a1e-11ea-9b05-b76e1d6895b6.png)

but HUNTINGBURG LAKE seems legit
```r
filter(LAKE_NAME == 'HUNTINGBURG LAKE') %>% select(CONTOUR, geometry) %>% plot()
```
![image](https://user-images.githubusercontent.com/2349007/78883476-2ca7f100-7a1f-11ea-9c21-f55e222c59dc.png)


and so does DEAM LAKE

![image](https://user-images.githubusercontent.com/2349007/78883532-3e899400-7a1f-11ea-937d-5de6ec8c5901.png)

**Comment by jread-usgs** [comment]
DEAM LAKE is `nhdhr_96568183` and HUNTINGBURG LAKE is `nhdhr_62689235` based on using `sf::st_crop` to the `sf::st_bbox` of the filtered lake in the contour dataset. NHDHR in red, this dataset in black:

![image](https://user-images.githubusercontent.com/2349007/78889110-adb7b600-7a28-11ea-8178-bab4f0f0d3bb.png)

**Comment by jread-usgs** [comment]
fixed in #169
